cg/store/models.py

```diff
@@ -181,10 +181,23 @@ class Application(Base):
     pipeline_limitations: Mapped[list["ApplicationLimitations"]] = orm.relationship(
         back_populates="application"
     )
-    order_types: Mapped[list["OrderTypeApplication"]] = orm.relationship(
-        "OrderTypeApplication", back_populates="application"
+    order_type_applications: Mapped[list["OrderTypeApplication"]] = orm.relationship(
```

Comment:
don't you need an alembic migration for the renaming of this attribute?

Reply:
Nope, relationships are only SQLAlchemy - the underlying database needs no change for this to work

Reply:
ok then nevermind